trigger/patch.go

```diff
@@ -173,7 +173,7 @@ func checkPatchStatus(p *patch.Patch) (bool, *patch.Patch, bool, error) {
 	// make sure the parent is done, if not, wait for the parent
 	if p.IsChild() {
 		if !evergreen.IsFinishedPatchStatus(parentPatch.Status) {
-			return isReady, nil, false, nil
+			return isReady, parentPatch, false, nil
```

Comment:
Should we also do this for 170, 181, 184?

Reply:
probably a good idea. done. thanks!